### DISCUSSION TITLE
11.3 + JclWin32.CreateMutex

### DISCUSSION THREAD

**Discussion by kazuser** [project-jedi/jcl#129] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/3335273/235054071-0aaf3dd7-6926-4129-b7f7-44a7ef196664.png)

**Comment by obones** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/1314739/235480853-a10fd819-9093-448e-b22c-5726d7628820.png)
